examples/images_contours_and_fields/pcolormesh_grids.py

```diff
@@ -0,0 +1,132 @@
+"""
+============================
+Pcolormesh grids and shading
+============================
+
+`.axes.Axes.pcolormesh` and `~.axes.Axes.pcolor` have a few options for
+how grids are laid out and the shading between the grid points.
+
+Generally, if *Z* is size *MxN* then the grid *X* and *Y* can be
```

Comment:
size -> shape (for an array t, t.shape is its shape, t.size its number of elements, so it's really not the same thing)  ("is size" -> "has shape"
(throughout)
I also prefer (M, N) to MxN but won't overly insist on it.

Reply:
I think I got all the shape/size changes...